src/mixxx.cpp

```diff
@@ -1229,6 +1229,13 @@ void MixxxMainWindow::initActions()
     m_pHelpManual->setWhatsThis(buildWhatsThis(manualTitle, manualText));
     connect(m_pHelpManual, SIGNAL(triggered()), this, SLOT(slotHelpManual()));
 
+    QString shortcutsTitle = tr("&Keyboard Shortcuts");
```

Comment:
rename to make it clear this launches in a browser.  "Show Keyboard Shortcuts in Browser..."

or "Show Shortcuts on Web..."  anyway, something with "..." at the end which indicates that a new window will be opened.